src/de.hpi.swa.graal.squeak.test/src/de/hpi/swa/graal/squeak/test/AbstractSqueakTestCase.java

```diff
@@ -43,11 +43,6 @@
     protected static SqueakImageContext image;
     protected static PointersObject nilClassBinding;
```

Comment:
I don't know where `nilClassBinding` comes from, where is this field set?

Reply:
It is set in AbstractSqueakTestCaseWithDummyImage, but it is declared in AbstractSqueakTestCase since it is used in the makeMethod... methods declared there.
All the makeMethod... methods are only called from subclasses of AbstractSqueakTestCaseWithDummyImage